### DISCUSSION TITLE
`amenity=fixme`

### DISCUSSION THREAD

**Discussion by habi** [Zverik/every_door#546] [General]
Several users (in Switzerland) have been generating `amenity=fixme`s which feel like they would be *much* better suited to be added to OSM as notes.
See https://www.openstreetmap.org/changeset/133126948 or https://www.openstreetmap.org/changeset/133250121 for examples.

Is that intended behaviour of EveryDoor, as the tag itself is not documented?

**Comment by habi** [comment]
I was not able to find a way to add `amenity=fixme` in EveryDoor 3.0 on iOS...

**Reply by deevroman** [reply]
<img width="456" alt="Снимок экрана 2023-03-14 в 17 35 39" src="https://user-images.githubusercontent.com/25708359/225035259-552e791d-b0c3-4aae-901d-3cea1c627fe9.png">

**Reply by habi** [reply]
Or, when doing a typo in searching (trst instead of test), then it leads to `amenity=fixme` + `fixme=type` + `fixme:type=trst` :)

**Comment by habi** [comment]
Worldwide there's now already 500+ undocumented `amenity=fixme`s: https://taginfo.openstreetmap.org/search?q=amenity%3Dfixme

**Comment by habi** [comment]
Here's a small MapRoulette challenge to pull all the `amenity=fixme` from the OSM data, geared to help fix those: https://maproulette.org/challenge/38449/
